src/governance-object.h

```diff
@@ -48,6 +49,9 @@ static const int SEEN_OBJECT_ERROR_IMMATURE = 2;
 static const int SEEN_OBJECT_EXECUTED = 3; //used for triggers
 static const int SEEN_OBJECT_UNKNOWN = 4; // the default
 
+static const int GOVERNANCE_FILTER_MAX_ELEMENTS = 20000;
```

Comment:
However maybe juggle governance-object.h to be like this:

static const int GOVERNANCE_FILTER_PROTO_VERSION = 60800;
static const double GOVERNANCE_FILTER_FP_RATE = 0.001;